src/groupeddataframe/groupeddataframe.jl

```diff
@@ -45,7 +45,11 @@ into row groups.
   ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR).
 - `sort` : whether to sort groups according to the values of the grouping columns
   `cols`; if all `cols` are `CategoricalVector`s then groups are always sorted
-  irrespective of the value of `sort`
+  irrespective of the value of `sort`; if `sort=false` then the order of groups
+  is undefined (a typical case is that they follow the order of appreance of
+  respecive values in the grouping columns, but a notable exception is when the
+  columns are `PooledVector`s, in which case they are ordered accoring to the `pool`
+  field in these vectors)
```

Comment:
I don't think that's true: in general the order is really undefined due to the dict-like grouping fallback. And the pool of `PooledDataArray` isn't exposed to users so it's not really useful to tell give them this information.

Reply:
I have updated the description to make it more precise.